topics/epigenetics/tutorials/hicexplorer/tutorial.md

```diff
@@ -0,0 +1,299 @@
+>
+>    > ### {% icon tip %} Tip: Importing data via links
+>    >
+>    > * Copy the link location
+>    > * Open the Galaxy Upload Manager
+>    > * Select **Paste/Fetch Data**
+>    > Below are the links to the read files that can be copied and pasted in the upload manager.
+>    >
+>    > ``` ftp://ftp.sra.ebi.ac.uk/vol1/fastq/SRR195/007/SRR1956527/SRR1956527_1.fastq.gz
+>    > ftp://ftp.sra.ebi.ac.uk/vol1/fastq/SRR195/007/SRR1956527/SRR1956527_2.fastq.gz
+>    > ftp://ftp.sra.ebi.ac.uk/vol1/fastq/SRR195/008/SRR1956528/SRR1956528_1.fastq.gz
+>    > ftp://ftp.sra.ebi.ac.uk/vol1/fastq/SRR195/008/SRR1956528/SRR1956528_2.fastq.gz
+>    > ftp://ftp.sra.ebi.ac.uk/vol1/fastq/SRR195/009/SRR1956529/SRR1956529_1.fastq.gz
+>    > ftp://ftp.sra.ebi.ac.uk/vol1/fastq/SRR195/009/SRR1956529/SRR1956529_2.fastq.gz
+>    >```
+>    >
+>    > * Paste the links into the text field
+>    > * Press **Start**
+>    {: .tip}
+>
+>    > ### {% icon tip %} Tip: Change the file type `fastq` to `fastqsanger` once the data file is in your history
+>    >
+>    > * Click on the pencil button displayed in your data file in the history
+>    > * Choose **Datatype** on the top
+>    > * Select `fastqsanger`
+>    > * Press **save**
+>    {: .tip}
+>
+> 3. Rename the dataset to something meaningful
+> By default, when data is imported via its link, Galaxy names it with its URL.
+>
+{: .hands_on}
+
+# Reads mapping
+
+Mates have to be mapped individually to avoid mapper specific heuristics designed for standard paired-end libraries.
+
+We have used the HiCExplorer sucessfuly with bwa, bowtie2 and hisat2. In this tutorial we will be using **Map with BWA-MEM** tool. It is important to remember to:
+- use local mapping, in contrast to end-to-end. A fraction of Hi-C reads are chimeric and will not map end-to-end thus, local mapping is important to increase the number of mapped reads
+- tune the aligner parameters to penalize deletions and insertions. This is important to avoid aligned reads with gaps if they happen to be chimeric.
+
+> ### {% icon hands_on %} Hands-on: Mapping reads
+>
+> 1. **Map with BWA-MEM** {% icon tool %}: Run Map with BWA-MEM on `SRR1956527_1` with:
+>    - "Will you select a reference genome from your history or use a built-in index?" to `Use a built-in genome index`
+>    - "Using reference genome" to `mm10`
+>    - "Single or Paired-end reads" to `Single`
+>    - "Select fastq dataset" to `SRR1956527_1`
+>    - "Select analysis mode" to `Full list of options`
+>    - "Set algorithmic options?" to `Set`
+>    - "Skip seeds with more than that many occurrences" to `10000`
+>    - "Set scoring options?" to `Set`
+>    - "Gap extension penalties; a gap of size k cost '-O + -E*k'. If two numbers are specified, the first is the penalty of extending a deletion and the second for extending an insertion" to `50,50`
+>    - "Penalties for 5'-end and 3'-end clipping" to `0`

```

Comment:
These are the settings for the wrong wrapper (Version 0.7.17.1) but we need the settings for 0.8.

Reply:
Use the following: `-A 1 -B 4 -E 50 -L 0` for BWA-MEM in version 0.8